modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/translation_manager/index.js

```diff
@@ -118,21 +118,23 @@ const TranslationManager = ({
 								setVisibleModal(true);
 							}}
 						/>
+						<div className="autofit-col">
```

Comment:
Same as my comment above, I believe there should be an empty line between these two elements. Not sure why SF didn't pick it up.